tbtaf/tbtaf_launcher.py

```diff
@@ -6,17 +6,9 @@
 from __future__ import absolute_import
 from __future__ import print_function
 from orchestrator.TBTAFOrchestrator import TBTAFOrchestrator
+from listener.TBTAFListener import Listener
 
 if __name__ == '__main__':
-    testScript = "./test/test05.tbtaf"
-    print("Welcome to TBTAF Test bed")
-    
-    myTBTAF = TBTAFOrchestrator()
-    
-    print("Executing the following test script: " + testScript)
-    
-    parseResult = myTBTAF.parseScript(testScript)
-
-    print("Parse Status: "+parseResult.status)
-
-    print("Parse Message"+parseResult.message)
+    listener = Listener() 
+    #myTBTAF = TBTAFOrchestrator()
+    listener.check_push_events()
```

Comment:
Please revist this considering the suggestions previously made for the APIs from the the listener